README.md

```diff
@@ -47,8 +47,8 @@ This implementation has the following known limitations:
 
 * In Ember versions before 3.11, modifiers cannot be passed to the element,
   even with addons such as the [modifier manager](https://github.com/ember-polyfills/ember-modifier-manager-polyfill)
-  and [on modifier](https://github.com/buschtoens/ember-on-modifier) polyfills
-  are used. Doing so requires [RFC #435](https://github.com/emberjs/rfcs/blob/master/text/0435-modifier-splattributes.md)
+  and the [on modifier](https://github.com/buschtoens/ember-on-modifier) polyfills.
+  Doing so requires [RFC #435](https://github.com/emberjs/rfcs/blob/master/text/0435-modifier-splattributes.md)
```

Comment:
I think the actual typo here is "even with addons such as" -> "even **when** addons such as", the the rest of the original sentence makes sense (to me... I think?)

Reply:
I agree @chancancode 👍

I've updated the PR